CHANGELOG.md

```diff
@@ -100,6 +100,7 @@ New deprecation(s):
 ### Other
 
 - **General**: Bump deps and k8s deps to 0.29.7 ([#6035](https://github.com/kedacore/keda/pull/6035))
+- **Cron Scaler**: Simplify code to determine next start and stop times ([#6056](https://github.com/kedacore/keda/pull/6056))
```

Comment:
Can you move this to unreleased, please? (after v2.15.1 it appears under it)

Reply:
Good catch... a bad merge by git. Yes, I will rebase and move it.

Reply:
please generate again the manifests too -> `make manifests`